### PR TITLE
Translate Chinese to follow the English's changes

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: content/doc/installation/from-source.zh-cn.md [hunk 1]
```diff
@@ -17,4 +17,68 @@ menu:
...
+```
+git tag -l
+git checkout v1.0.0
+```
+
+## 编译
+
+我们已经将所有的依赖项拷贝到本工程，我们提供了一些 [编译选项](https://github.com/go-gitea/gitea/blob/master/Makefile) 来让编译更简单。你可以按照你的需求来设置编译开关，可用编译选项如下：
+
+* `bindata`: With this tag you can embed all assets required to run an instance of Gitea, this makes a deployment quite easy because you don't need to care about any additional file.
```
Comment (tboerger) [inline content/doc/installation/from-source.zh-cn.md]: Missed that?

**Review by tboerger — COMMENTED** [review]
Just a single question, otherwise trusted LGTM from me, I can't read chinese :)

File: content/doc/installation/from-source.zh-cn.md (hunk 1 above)
Comment (lunny) [reply content/doc/installation/from-source.zh-cn.md]: yes. I will send another commit.

**Comment by andreynering** [comment]
LGTM

**Comment by tboerger** [comment]
Missing the other commit

**Comment by tboerger** [comment]
Trusted LGTM

**Comment by lunny** [comment]
I will send a PR to sync English version to this one after this LGTM

**Comment by appleboy** [comment]
LGTM

**Comment by appleboy** [comment]
Waiting merge zh-cn translations. I will start translate zh-cn to zh-tw. Thanks.

**Comment by lunny** [comment]
Drone?

File: content/doc/installation/from-source.zh-cn.md [hunk 2]
```diff
@@ -17,4 +17,68 @@ menu:
...
+
+* Go 工具
+
+使用 Go 工具编译需要你至少安装了Go 1.5以上版本并且将 govendor 的支持打开。执行命令如下：
+
+```
+go build -tags="bindata"
+```
+
+如果要使用bindata选项，你可能需要先执行go generate
```
Comment (tboerger) [inline content/doc/installation/from-source.zh-cn.md]: A simple go generate will also call generate for vendor. It requires our packages as a parameter without the makefile. I suggest to document only the make process.

**Comment by lunny** [comment]
LGTM work?

**Pull request merged**